test/e2e/tests/network/add-custom-network.spec.js

```diff
@@ -454,30 +454,27 @@ describe('Custom network', function () {
             text: 'Add',
           });
 
-          const [currencySymbol, networkUrl] = await driver.findElements(
-            '.definition-list dd',
-          );
-          assert.equal(
-            await currencySymbol.getText(),
-            currencySYMBOL,
-            'Currency symbol is not correctly displayed',
-          );
-          assert.equal(
-            await networkUrl.getText(),
-            networkURL,
-            'Network Url is not correctly displayed',
-          );
+          await driver.waitForSelector({
+            tag: 'dd',
+            text: currencySYMBOL,
+          });
```

Comment:
Could you add a more specific data-testid for this element or make be some other approach? 
The reason I mention this is - I changed the text for currencySYMBOL from ETH to ET, the test would not fail leading to false negatives. 
I also checked for any other unique identifiers but there aren't any so we need to add one in the code.

Reply:
Update on my previous comment - I think changing the data-testid would not work too.

Reply:
nice catch! I've added test ids in the app level :pray: